sockeye/translate.py

```diff
@@ -87,6 +91,12 @@ def run_translate(args: argparse.Namespace):
                            'This feature may be removed or change its behaviour in future. '
                            'DO NOT USE IT IN PRODUCTION!')
 
+        # Skip softmax for a single model when not doing ensembling or beam search or sampling
+        skip_softmax = False
```

Comment:
please rebase onto latest master with skip_softmax changes added

Reply:
Done. I changed the function to pass `skip_softmax` directly, since it now needs to check both `not args.sample and not output_handler.reports_score()` (and passing just one variable seems better).